src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/DataStoreApiClient.java

```diff
@@ -34,6 +28,16 @@
     String CASE_ID = "caseId";
     String CASE_TYPE_ID = "ctid";
 
+    String CASES_WITH_ID = "/cases/{caseId}";
+
+    String SEARCH_CASES = "/searchCases";
+    String INTERNAL_SEARCH_CASES = "/internal/searchCases";
+    String CASE_USERS = "/case-users";
+    String INTERNAL_CASES = "/internal/cases/{caseId}";
+    String START_EVENT_TRIGGER = "internal/" + CASES_WITH_ID + "/event-triggers/{eventId}";
```

Comment:
This results in URL with too many slashes - `internal//cases/{caseId}/event-triggers/{eventId}`

Change to 

`String START_EVENT_TRIGGER = INTERNAL_CASES + "/event-triggers/{eventId}";`

Reply:
Seen resolved in 1021e05.